app/models/staged/petition_creator/has_creator_signature.rb

```diff
@@ -0,0 +1,30 @@
+module Staged
+  module PetitionCreator
+    module HasCreatorSignature
+      extend ActiveSupport::Concern
+
+      included do
+        def valid?
+          super && creator_signature_valid?
```

Comment:
This uses short circuiting and does not validate creator's signature if ```super``` returns ```false```, we potentially lose the display of some error msgs; should it use eager evaluation instead?